task/github.py

```diff
@@ -329,25 +329,15 @@ def issues(self, labels=["bot"], state="open", since=None):
         result = []
         page = 1
         count = 100
-        opened = True
         label = ",".join(labels)
-        while count == 100 and opened:
-            req = "issues?labels={0}&state=all&page={1}&per_page={2}".format(label, page, count)
+        while count == 100:
+            req = "issues?labels={0}&state={1}&page={2}&per_page={3}".format(label, state, page, count)
```

Comment:
This certainly makes sense to leave the state filtering to the github server side. Maybe that was some workaround for traversing through closed states with the paging, but hard to tell -- if that works now, let's by all means use it.